tools/coverage_system_tests.rc

```diff
@@ -1,12 +1,26 @@
+; This file controls what modules are examined for code coverage in our system tests.
+
 [run]
 omit =
+    ; Common code, so covered by nifake unit tests.
+    ; Some modules don't even exercise all of _attributes.py and _converters.py, making parts unreachable in system tests.
     */__init__.py
     */_attributes.py
     */_converters.py
+    */errors.py
     */_library_singleton.py
     */_visatype.py
+
+    ; Although the enum names and values vary between modules, all enums are mostly the same. nifake unit tests cover the various kinds that we use.
     */enums.py
-    */errors.py
-    */session.py
 
+    ; Although there is unit test coverage for session.py (in nifake, nidigital, nimodinst, nitclk), that doesn't cover all of our hand-coded fancy functions, so don't omit it.
```

Comment:
Why even mention anything about session.py? 
The way I see it, if it's not here, then it's covered right?

Is it because you are wanting to explain the rationale? I don't think it's necessary.